src/base/orgs/View.svelte

```diff
@@ -17,6 +17,7 @@
   import { Profile, ProfileType } from '@app/profile';
   import Projects from './View/Projects.svelte';
   import Link from '@app/Link.svelte';
+  import SeedId from '@app/SeedID.svelte';
```

Comment:
Maybe name the file `SeedId` as well

Reply:
Actually it's called a "SeedAddress", the ID is just the PeerID without the hostname.